docker-app/qfieldcloud/subscription/models.py

```diff
@@ -882,7 +882,11 @@ def create_subscription(
         return trial_subscription_obj, regular_subscription_obj
 
     def __str__(self):
-        return f"{self.__class__.__name__} #{self.id} user:{self.account.user.username} plan:{self.plan.code} total:{self.active_storage_total_mb}MB"
+        plan_code = self.plan.code if hasattr(self, "plan") else "None"
```

Comment:
I don't think there is a situation that a subscription can have no plan, as it is a non-nullable field. In short, this line is unneeded extra.

Reply:
Ok, I'll take your word for it!